history-of-quakers/modernized/23-chapter-21.adoc

```diff
@@ -424,6 +456,10 @@ which grew jealous that by the repeal of those laws, the Catholics,
 who now had a king of their own religion, would become too powerful.
 In the meanwhile the king published the following proclamation:
 
+[.embedded-content-document.legal]
+--
+
+[.letter-heading]
 James R.
```

Comment:
I forgot how we did this in the other PR. This looks like a salutation or letter heading, but I think it's from the king, written by someone else.

Reply:
Yeah, for the last one I think we moved it down to become a signature, but this one already has a signature... 🤔

How about we change it to a `[.signed-section-context-open]` -- it seems to sort like a stamp saying "on behalf of James R." so maybe that's not to much of a stretch.